api/api/controllers/search_controller.py

```diff
@@ -170,19 +171,23 @@ def get_excluded_providers_query() -> Q | None:
     Hide data sources from the catalog dynamically.
     To exclude a provider, set ``filter_content`` to ``True`` in the
     ``ContentProvider`` model in Django admin.
+    The list of ``provider_identifier``s is cached in Redis with
+    `:1:FILTER_CACHE_KEY` key.
     """
 
-    filter_cache_key = "filtered_providers"
-    filtered_providers = cache.get(key=filter_cache_key)
+    filtered_providers = cache.get(key=FILTERED_PROVIDERS_CACHE_KEY)
     if not filtered_providers:
-        filtered_providers = models.ContentProvider.objects.filter(
-            filter_content=True
-        ).values("provider_identifier")
+        filtered_providers = list(models.ContentProvider.objects.filter(filter_content=True).values_list(
+            "provider_identifier",
+            flat=True,
+        ))
```

Comment:
This should fix the linting:
```suggestion
        filtered_providers = list(
            models.ContentProvider.objects.filter(filter_content=True).values_list(
                "provider_identifier", flat=True
            )
        )
```